code/drasil-docLang/Drasil/DocDecl.hs

```diff
@@ -133,7 +133,7 @@ mkDocDesc SI{_inputs = is, _sysinfodb = db} = map sec where
   scsSub Assumptions = DL.Assumptions $ fromConcInsDB assumpDom
   scsSub (TMs s f) = DL.TMs s f $ allInDB theoryModelTable
   scsSub (GDs s f dd) = DL.GDs s f (allInDB gendefTable) dd
-  scsSub (DDs s f dd) = DL.DDs s f (allInDB dataDefnTable) dd
+  scsSub (DDs s f dd) = DL.DDs s f (allInDB eDataDefnTable) dd -- TODO: What about the ModelExpr ones?
```

Comment:
Well, I suspect it's around this area that's causing the `DataDefinition ModelExpr`s` to not appear. I'm not quite sure how to fix it yet however, still researching it.